core/modules/nfv_monitor.h

```diff
@@ -57,7 +63,7 @@ class NFVMonitor final : public Module {
   std::unordered_map<Flow, uint64_t, FlowHash> per_flow_packet_counter_;
 
   // Core statistics
-  std::vector<uint64_t> per_core_latency_sample_;
+  boost::circular_buffer<uint64_t> per_core_latency_sample_;
```

Comment:
IIRC, using boost library requires a installation process. I wonder if you could help automate the process in build.py. Thanks

Reply:
Yes it requires an installation. For bess the dependencies are simply listed on the wiki. Will look into automating this via build.py

Reply:
The boost libraries currently are installed via apt-get. Using apt-get in the build script will make it system dependent as some systems may have other package managers. Another way to install boost in a system independent way is to download and build it from source. However I feel that will take a lot more effort and this is not the goal right now. Anyways the build script does not install any of the dependencies and they are just listed on the wiki page.

Reply:
Make senses.
Can you modify this page 'https://github.com/NetSys/bess/wiki/Build-and-Install-BESS' to help users install the correct set of dependencies. I'd say do it in your next PR. Thanks.